### ISSUE TITLE
New endpoint - Get Channel Stream Schedule

### ISSUE THREAD

**Issue by xeon927** [Plazide/node-twitch#34]
Were there any plans to implement the new(ish) endpoint as per https://dev.twitch.tv/docs/api/reference#get-channel-stream-schedule ?

If needed, I can have a poke around on my machine and see if I can get it to work based on the other existing endpoints as a guide?

**Comment by Plazide** [comment]
Yeah, the goal is to support every endpoint. I just don't have time to work on this as much anymore. 

You are very welcome to try it yourself. The most tedious part is adding the types for the method. You need to add at least three types, one in `src/types/objects.ts`, one in `src/types/responses.ts` and on in `src/types/options.ts`.

You then need to add tests for the new method in `tests/api.test.ts`.

Let me know if  you have any questions or there is anything you are unsure about, either through a comment in this issue or via a DM on Twitter([@chj_web](https://twitter.com/chj_web)).
